docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,253 @@
+---
```

Comment:
This doc is **very similar** to the ["Get started" tutorial](https://docs.microsoft.com/en-us/dotnet/spark/tutorials/get-started), which also explains how to prepare a Windows environment for .NET for Apache Spark. Could we just link to the first part of that document rather than repeating it here? Or if we do want to repeat it, we should explain the steps in the same wording/format as we do in that doc to be consistent.

Reply:
The "Get started" tutorial has mostly same steps, except it is missing installation of Apache Maven. I can change the wording/format in this doc to match that to be consistent.